vars/common-vars.yml

```diff
@@ -47,3 +48,11 @@ suse_airship_components_image_tag: "{%- if suse_airship_build_local_images or su
                                     {%- else -%}
                                       latest
                                     {%- endif -%}"
+suse_osh_image_version: "pike"
```

Comment:
Does it make sense to set this differently than the value in playbooks/roles/deploy-osh/defaults/main.yml ?

Reply:
Good question. I see that playbooks/roles/deploy-osh/defaults/main.yml uses ocata. Is Ocata osh only deployment actually deploys? on Ubuntu or SUSE? 

Airship deployment has been using Pike for a quite a while and works pretty well for us. If the osh squad only uses Ocata for Ubuntu, can I suggest we align on Pike? This is also what demo at SUSECON, so I really don't think we want to use Ocata.

I cam imagine OSH squad could work on a newer version of openstack before it is ready for the Airship squad to adopt. Actually that's the case now, since you are on Rocky. So maybe we can add an override in OSH to the common var value? I am not against either to the idea Itxaka mentioned below to use a different variable. Whatever works.

Reply:
My main problem with this is that there is the same var name in 2 places which can be set to different values, leading to problems when running this, so either we sync on the same branch (rocky for OSH currently) or we set different names as to not confuse ourselves :+1:

Reply:
I would thin it makes sense to use same release, but if it is not  possible for some time, let's have different variable names...

Reply:
yeah I think we should have them separated for the moment but one next to the other in `vars/common-vars.yml` so we can merge them in the future and we dont forget about them

Reply:
I am fine with different version number for now. let me push a new commit. I'll keep "suse_osh_image_version" for deploy_osh role, and have a new var name "suse_airship-openstack_helm_image_version" for the airship deployment.

Reply:
actually, I am going to back out the change that adds the osh image version in the common vars. This can/should be in a separate commit as it is not required for the UCP site refactoring. And we will talk about convergence in theinter-squad meeting.

Reply:
sounds good to me